projects/plugins/jetpack/changelog/add-zendesk-chat-authentication

```diff
@@ -0,0 +1,4 @@
+Significance: major
```

Comment:
The "major" changelog here bumped Jetpack Protect to `2.0.0`, which I think is unintended here (same for the other plugins).

Reply:
I also didn't notice that when I was working on and reviewing the PR. :/

Reply:
I believe it should just be a matter of editing the changelogs down to "patch" and then re-running `./tools/fixup-project-versions.sh` :+1:

Here's what I did for Protect: https://github.com/Automattic/jetpack/pull/31952

^ Includes adjustments to some other changelogs as well, you can ignore those.

Reply:
Ok I will open up a PR for this

Reply:
It seems like the only major version updates were Protect, Search, Migration, and Social. You fixed Protect already, it seems like I can fix Search and Migration the way you described. Social seems to have had further updates since then and I think? already been updated to 2.0 🤔 

Anyway here is the PR: https://github.com/Automattic/jetpack/pull/31978

@IanRamosC could you take a look at that one?